composables/masto/statusDrafts.ts

```diff
@@ -112,9 +112,14 @@ export function getReplyDraft(status: mastodon.v1.Status) {
   }
 }
 
-export function isEmptyDraft(draft: Draft | null | undefined) {
+export function isEmptyDraft(drafts: Array<DraftItem> | null | undefined) {
+  if (!drafts)
+    return true
+
+  const draft = drafts.at(0)
```

Comment:
This line raises an exception if we have some existing draft value in the local storage written before this change.

I think this is due to the change in the data structure of the draft.

Here's the example `elk-drafts` value:

![image](https://github.com/elk-zone/elk/assets/1425259/b412df4c-5a27-4122-9238-9a16b1c0208d)

As you can see, the old structure (Draft object) and the new one (array of Draft object) are mixed. Is it possible to handle both old and new values to ensure backward compatibility?

Reply:
Oh yes good find! I wanted to, but totally forgot testing it with existing drafts. Will try to make it compatible

Reply:
I think I fixed it, but please feel free to double check with your real examples to be sure 👍🏻 

I also figured out some reactivity issues with drafts, which should now be fixed as well

Reply:
Thanks! I confirmed the fix now 👍🏻